utt/admin-cli/include/admin.hpp

```diff
@@ -33,7 +33,7 @@ class Admin {
   /// [TODO-UTT] Should be performed by an admin app
   /// @brief Deploy a privacy application
   /// @return The public configuration of the deployed application
-  static bool deployApp(Channel& chan, bool budget_policy = true);
+  static bool deployApp(Channel& chan, bool budget_policy);
```

Comment:
Should this PR also include a change to the wallet app?
as it checks 
```
const size_t budget = getPrivacyBudget();
  if (budget < amount) throw std::runtime_error("User has insufficient privacy budget!");```

Reply:
Isn't it more elegant to define the non-budget configuration as a budget coin with infinite value?
It will eliminate the need for `if - else` on the transaction and will reduce the configuration changes

Reply:
It's more elegant for the code but will it be more elegant for the system if we still need to create and manage a 'hidden infinite' budget coin? And how does it reduce the configuration changes?